.dockerignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
Maybe you want also to add `.git` directory?

Reply:
Thanks! Added :)